tutorials/3d/spatial_material.rst

```diff
@@ -462,6 +462,38 @@ UV or triplanar modes, many interesting textures can be achieved.
 
 .. image:: img/spatial_material24.png
 
+There are several settings that control how detail is used.
+
+Mask: The detail mask is a black and white image used to control where the
+blending takes place on a texture. White is for the detail textures, Black
+is for the regular material textures, different shades of gray are for
+partial blending of the material textures and detail textures.
+
+Blend Mode: These four modes control how the textures are blended together.
+
+- Mix: Adds pixel values of one texture with the other. Neither texture is
+  completely mixed at white or black, instead being completely mixed in gray
+  parts of a detail mask.
```

Comment:
I am having trouble understanding what you mean by "neither texture is completely mixed" maybe something like this?
```suggestion
- Mix: Combines pixel values of both textures. At black, only show the material texture,
  at white, only show the detail texture. Values of gray create a smooth blend between
  the two.
```